message ssl_planner_packet {
	optional string comment = 1;
}
